deploy/paas/README.md

```
@@ -0,0 +1,31 @@
+# Docs — PaaS deployment
+
+Buildpack scripts and nginx template for deploying [Docs](https://github.com/suitenumerique/docs) on a PaaS (Scalingo, Clever Cloud, …).
+
+## Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)